2024.5 iOS blog post - Add extra request for feedback (#32884)
home-assistant/home-assistant.github.io: source/_posts/2024-05-22-companion-app-for-ios-20245-assist.markdown

- We also added a button to reload your widgets, since sometimes they may become blank for a few users after an update. For actions, we added a button to update server-created Actions manually, so you don't need to reopen the app for that (but remember to restart HA after adding or removing Actions from YAML).

> - Memory usage and performance issues on our macOS client are big topics in our GitHub right now, and we are aware of them and are working towards improving them. This release is the first iteration in that direction, we ask you to send your feedback since this issue is not easily reproducible in all instances.

Reviewer: ```suggestion
- Memory usage and performance issues on our macOS client are big topics in our GitHub right now, and we are aware of them and are working towards improving them. As this release is the first iteration in that direction, we would like your feedback since this issue is not easily reproducible in all instances.
```
